src/main/java/com/google/idea/perf/TracerView.kt

```diff
@@ -90,13 +95,20 @@ class TracerView(parentDisposable: Disposable) : JBPanel<TracerView>() {
         // Progress bar.
         progressBar = JProgressBar().apply {
             isVisible = false
-            maximumSize = Dimension(Integer.MAX_VALUE, minimumSize.height)
+            maximumSize = maximumChildSize
         }
         add(progressBar)
 
         // Call list.
         add(JBScrollPane(listView))
 
+        // Render time label.
+        refreshTimeLabel = JBLabel()
```

Comment:
Currently there is a lot off "jitter" as the label text is updated, I think either because the number of decimal digits changes over time or because the font is not mono-space.

Reply:
Yeah, this could be an issue with variable-spaced fonts if the number of digits in the label changes. I noticed that the refresh time shifts between 100s-1000s of microseconds. Should I just go for a mono-space font with whitespace padding?

Reply:
Yes, a mono-space font sounds good to me.